frontend/src/libs/components/header/header.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import styles from './header.module.scss';
+
+type Properties<T> = T;
```

Comment:
```suggestion
type Properties = {}; // describe props that you need
```

Reply:
> should looks like this <img alt="image" width="1079" src="https://user-images.githubusercontent.com/42296182/225271598-ef659762-b2db-4c7f-be2f-f3dd3e3fe2a8.png">

I’m not sure I fully understand how much I should do in my task. As I see in the figma layout, the header’s styles should vary depending on which route we are on, so do I need to cover all style cases or should my component only have the same styles as the header you attached in the comment?